subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -229,3 +229,31 @@ Run with `--watch-fs` on the command line::
 This enables watching the file-system for this build only.
 Put `org.gradle.unsafe.watch-fs=true` in your `gradle.properties`::
 This enables watching the file-system for all builds, unless explicitly disabled with `--no-watch-fs`.
+
+[[sec:daemon_watch_fs_troubleshooting]]
+=== Troubleshooting file-system watching
+
+Limitations::
+- If your build doesn’t declare inputs and outputs properly, there is a chance Gradle does not detect changes to undeclared stuff.
```

Comment:
This isn't a limitation. I'd mention this under the "Gradle doesn't pick up my changes" section below.

Reply:
I removed the point, since it is already mentioned under "Gradle doesn't pick up my changes".